src/common/k8s-api/kube-api.ts

```diff
@@ -414,7 +409,7 @@ export class KubeApi<
         return await this.getLatestApiPrefixGroup();
       } catch (error) {
         // If valid API wasn't found, log the error and return defaults below
-        this.logger.error(`[KUBE-API]: ${error}`);
+        logger.error(`[KUBE-API]: ${error}`);
```

Comment:
Hmm. Why is this explicit side-effect restored?

Reply:
hahaha, yes actually because these were making behavioural unit tests fail :(. 

I will try do this differently.